### PR TITLE
[13.0][FIX] stock_return_request: missing cache

### PR DESCRIPTION
**Pull request by chienandalu**
Ensure that all the records get their value

cc @Tecnativa TT41230

please review @pedrobaeza @victoralmau

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]

File: stock_return_request/models/stock_move.py [hunk 1]
```diff
@@ -24,6 +24,7 @@ class StockMove(models.Model):
     def _compute_qty_returnable(self):
         """Looks for chained returned moves to compute how much quantity
            from the original can be returned """
+        self.qty_returnable = 0
```
Comment (rousseldenis) [inline stock_return_request/models/stock_move.py:27]:
```suggestion
        moves_to_update = self.filtered(lambda x: x.state not in ["draft", "cancel"])
        (self - moves_to_update).qty_returnable = 0
        for move in moves_to_update:
        (...)
```

Comment (rousseldenis) [reply stock_return_request/models/stock_move.py:27]: This avoid to trigger update on the whole recordset if not necessary

Comment (pedrobaeza) [reply stock_return_request/models/stock_move.py:27]: AFAIK, since v13, this is not a problem, as this doesn't retrigger a recomputation.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**